cmd/mdatagen/internal/loader_test.go

```diff
@@ -33,7 +33,7 @@ func TestLoadMetadata(t *testing.T) {
 						component.StabilityLevelBeta:        {"traces"},
 						component.StabilityLevelStable:      {"metrics"},
 					},
-					Distributions: []string{},
+					Distributions: nil,
```

Comment:
@yurishkuro I saw this change was made on https://github.com/open-telemetry/opentelemetry-collector/pull/10260. Do you know why?

Reply:
don't recall, probably was failing somewhere

Reply:
if you don't need it don't include it